tests/php/Core/Manifest/VersionProviderTest.php

```diff
@@ -15,6 +16,12 @@ class VersionProviderTest extends SapphireTest
      */
     protected $provider;
 
+    protected function setup(): void
+    {
+        parent::setup();
+        $this->clearCache();
```

Comment:
This was added because caching on VersionedProvider started working, in CMS 4 it is broken.

I've raised a CMS 4 bug issue https://github.com/silverstripe/silverstripe-framework/issues/10490

Reply:
Why do we need to clear cache on setup? That feels like something `SapphireTest` should be handling behind the scenes.

Reply:
I thought so too.  But apparently not.

Manually clearing caches is done in elsewhere in other unit tests e.g. https://github.com/silverstripe/silverstripe-assets/blob/1/tests/php/Shortcodes/ImageShortcodeProviderTest.php#L178

Reply:
Weird. I've created [a new card](https://github.com/silverstripe/silverstripe-framework/issues/10492) about that since it looks like a bug for me, but I'm okay with the workaround for this PR.